src/components/workshop-code.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import CardIcon from "@/components/ui/card-icon";
+import { cn } from "@/lib/utils";
+import { useState } from "react";
+import { useToast } from "@/components/ui/use-toast";
+
+export const workshopCodeLiteral = "DKEEH";
+
+export function WorkshopCode() {
+  const [codeCopied, setCodeCopied] = useState(false);
+  const { toast } = useToast();
```

Comment:
This also can be imported directly like so:
```ts
import { toast } from "@/components/ui/use-toast";
```
The hook is actually not needed (and moving forward I've been using the direct import - I would recommend doing it this way in the future).